Resources/config/intl.xml

```diff
@@ -6,6 +6,7 @@
 
     <parameters>
         <parameter key="sonata.intl.locale_detector.request.class">Sonata\IntlBundle\Locale\RequestDetector</parameter>
+        <parameter key="sonata.intl.locale_detector.request_stack.class">Sonata\IntlBundle\Locale\RequestStackDetector</parameter>
```

Comment:
I understand, but removing this is already a BC break so no need to had some hard work here. :-)